src/main/resources/i18n/outdoor/common.properties

```diff
@@ -359,8 +359,10 @@ UPDATING_ERROR_TITLE=Failed to update the order
 500_ERROR_TITLE=500-Internal Server Error
 500_ERROR_CONTENT=Something wrong in this page, please try again later or contact administrator.
 KEYCLOAK_ERROR_TITLE=Keycloak Unavailable
+KEYCLOAK_UNAUTHORIZED_CLIENT_TITLE=Unauthorized Client
 KEYCLOAK_SERVER_IS_NOT_AVAILABLE=Can't establish connection with Keycloak server.
 KEYCLOAK_REALM_NOT_FOUND=Keycloak realm not found from URI:
+KEYCLOAK_CLIENT_UNAUTHORIZED=Invalid client or invalid client credentials. Please check Keycloak configuration.
```

Comment:
Maybe change to: 
Invalid client secret is provided. Please check Keycloak configuration.